### PR TITLE
Create basic UI

### PR DESCRIPTION
**Pull request by Lembani**
> Highlights:
- Created a app.rb file that will serve as your console app entry-point. It has methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- Defined the entry point, this will be a method called `main` that is invoked at the end of the main.rb file. This method does the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: rental.rb [hunk 1]
```diff
@@ -5,12 +5,13 @@
...
-    @book = book
-    @person = person
 
+    @book = book
     book.rentals << self
+
+    @person = person
     person.rentals << self
   end
 end
```
Comment (Munsa1) [inline rental.rb]: - [ ] I am currently unable to create a rental. Check line 57 in app.rb 
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Lembani, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: rental.rb (hunk 1 above)
Comment (Lembani) [reply rental.rb]: Hi @Munsa1 , thanks for the feedback.
Let me fix the issue. 🙏🏾

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Lembani 

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lembani** [comment]
Thanks @omar-labana for approving my PR. I really appreciate.

**Pull request merged**